src/pages/Vote.tsx

```diff
@@ -4,19 +4,23 @@ import { Document } from '@contentful/rich-text-types'
 import renderRichText from 'src/util/renderRichText'
 import icon from '../images/icon.svg'
 import BasicPage from 'src/components/BasicPage'
+import { useTranslation } from 'react-i18next'
 
 export default function Vote() {
+  const { t } = useTranslation()
+
+  // TODO: Handle translation
```

Comment:
I didn't realize y'all were using a CMS. I wonder if [adding a locale in Contentful](https://www.contentful.com/help/working-with-translations/) will magically work here 😅 . My guess is it probably reads the browser's locale so we'd have to play around with configs to get it to read from local storage.